.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -502,6 +520,20 @@ slack-notify-on-push-build:
     JIRA_PROJECT_ID: $JIRA_PROJECT_ID
     JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME: $JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME
 
+slack-notify-on-merge:
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH == $master_branch_name'
```

Comment:
Why not always?

Reply:
same as the on push comment, i splitted the notify to 2 separate steps. each one handles another scenarion